src/composables/default-ref.ts

```diff
@@ -0,0 +1,22 @@
+import { computed, ref, UnwrapRef } from '@nuxtjs/composition-api'
+
+const NotSet = Symbol('NotSet')
+
+/**
+ * Produces a `computed` that returns a default value until
+ * an explicit value is set.
+ *
+ * @param getDefault - A function returning the default value
```

Comment:
We should also document the `@returns`.

Reply:
That's true. I guess it's not really important when the description is so detailed, but I always write it (even if twice).